data/campaigns/Eastern_Invasion/scenarios/02_The_Escape_Tunnel.cfg

```diff
@@ -297,8 +320,25 @@
 
         [message]
             speaker=Dacyn
-            message= _ "Very well. The best of luck in your battle."
+            message= _ "Very well. The best of luck in your battle. Gweddry, we may be able to avoid the bulk of the trolls by going through the dwarves' base."
```

Comment:
Use a Unicode quote (`’`).

Not to nitpick, but is "base" the right word here? Maybe "outpost"?

Reply:
I'd say it is, because this seems to be the only place where dwarves are living.
But tbh, I think 'dwarvish tunnel' fits even better.